lib/matplotlib/patches.py

```diff
@@ -4504,6 +4506,10 @@ def _get_xy(self, x, y, s, axes=None):
             #(0,0) is lower left, (1,1) is upper right of axes
             trans = axes.transAxes
             return trans.transform_point((x, y))
+        elif hasattr(s, "transform_point"):
```

Comment:
Should this be a tighter check for of `isinstance(s, matplotlib.transforms.Transform` instead? That way we make sure that it's a transform object.